### PR TITLE
Add rake tasks to copy staging configs to all staging environments.

### PR DESCRIPTION
**Pull request by dandrabik**
## WHAT
Add a rake task that allows you to:
1. Set a config and value for all staging environments
2. Remove a config from all staging environments
3. Copy a config on main staging to the rest of the staging environments
## WHY
We have a lot of staging environments and are changing configs a good amount. That can lead to broken environments, manual fixes, or just drift from the correct settings.
## HOW
Simple wrapper around the heroku commands.
### Screenshots
Screen Shot 2022-08-22 at 12 04 39 PM

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  No, just going to write rake tasks from now on and live on the edge.
Have you deployed to Staging? |  No deploy, but I set and unset staging vars
Self-Review: Have you done an initial self-review of the code below on Github? | Yes.
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | N/A 
Back-to-school 2022: Have you checked the [webinar schedule](https://www.notion.so/quill/Back-to-school-webinar-banners-2022-a75a89cfad9f434899ef6be3eb184733) to avoid for downtime/risky deploys? | N/A

### CODE REVIEW

**Review by anathomical — APPROVED** [review]
This looks good to me, and I agree that we're doing this often enough that we should have a tool like this.  One super-minor thing to note: the screenshot looks like it's the signature of `set_all`, but it says it's a call to `set_all`.  Probably just a run of a different version of the code, so no big deal.

**Comment by dandrabik** [comment]
> This looks good to me, and I agree that we're doing this often enough that we should have a tool like this. One super-minor thing to note: the screenshot looks like it's the signature of `set_all`, but it says it's a call to `set_all`. Probably just a run of a different version of the code, so no big deal.

I'm not sure I understand your comment (maybe there is a typo?). The screenshot should be accurate for running the rake task to copy from main staging to the rest of the staging apps.

File: services/QuillLMS/lib/tasks/staging_config.rake [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    STAGING_APP = 'empirical-grammar-staging'
+    STAGING_PERSONAL_APPS = %w(
+      quill-lms-brendan
+      quill-lms-cissy
+      quill-lms-dan
+      quill-lms-emilia
+      quill-lms-eric
+      quill-lms-pkong
+      quill-lms-thomas
+      empirica-grammar-security
```
Comment (emilia-friedberg) [inline services/QuillLMS/lib/tasks/staging_config.rake:78]: I think this is a typo (should be `empirical-grammar-security`)

**Review by emilia-friedberg — APPROVED** [review]
This is great. One small note on what I think is a typo but otherwise LGTM. Thanks for doing that.

**Comment by anathomical** [comment]
> > This looks good to me, and I agree that we're doing this often enough that we should have a tool like this. One super-minor thing to note: the screenshot looks like it's the signature of `set_all`, but it says it's a call to `set_all`. Probably just a run of a different version of the code, so no big deal.
> 
> I'm not sure I understand your comment (maybe there is a typo?). The screenshot should be accurate for running the rake task to copy from main staging to the rest of the staging apps.

I think I figured it out.  The last character is, I suspect, cut off in the screenshot.  So your example command ends with `\[ZZZ_TEST_KEY\]`, and that final slash is not a new-line enabler of some sort.  In turn, `dancedance123` is output from the script, not part of the input.  Sorry about the confusion.

File: services/QuillLMS/lib/tasks/staging_config.rake (hunk 1 above)
Comment (dandrabik) [reply services/QuillLMS/lib/tasks/staging_config.rake:78]: Good catch! 

Is is a typo, but it's a typo in the name of the app (I copied the list from a CLI command)
Screen Shot 2022-08-22 at 12 53 52 PM

**Comment by dandrabik** [comment]
> > > This looks good to me, and I agree that we're doing this often enough that we should have a tool like this. One super-minor thing to note: the screenshot looks like it's the signature of `set_all`, but it says it's a call to `set_all`. Probably just a run of a different version of the code, so no big deal.
> > 
> > 
> > I'm not sure I understand your comment (maybe there is a typo?). The screenshot should be accurate for running the rake task to copy from main staging to the rest of the staging apps.
> 
> I think I figured it out. The last character is, I suspect, cut off in the screenshot. So your example command ends with `\[ZZZ_TEST_KEY\]`, and that final slash is not a new-line enabler of some sort. In turn, `dancedance123` is output from the script, not part of the input. Sorry about the confusion.

Ah, I see, yeah, I should have not cropped so close to the edge. And for rake tasks using parameters the form is:

`rake namespace:command[param1, param2]`
 
but in `.zsh` you have to escape the `[` and `]`, so you end up with this less pretty version:

`rake namespace:command\[param1, param2\]`

**Pull request merged**